actionpack/lib/action_controller/railtie.rb

```diff
@@ -24,7 +24,7 @@ class Railtie < Rails::Railtie #:nodoc:
     initializer "action_controller.parameters_config" do |app|
       options = app.config.action_controller
 
-      ActiveSupport.on_load(:action_controller) do
+      ActiveSupport.on_load(:action_controller_base) do
```

Comment:
Yes you are right, it doesn't cover `ActionController::API`.

@rafaelfranca where do you see that it is executed twice? Is the same hook registered twice?